### DISCUSSION TITLE
Optifine Installation (1.16.5)

### DISCUSSION THREAD

**Discussion by KitsuneDev** [SKCraft/Launcher#454] [I need help!]
Hello, I am trying to bundle Optifine with my Forge 1.16.5 Modpack. However, I am currently unable to achieve my desired result. Are there any instructions on how to do so?

**Comment by Nydid** [comment]
Optifine does not want anyone to host their own version of the mod. But if you get the link to the Optifine .jar download, you can make the launcher download it. The only issue is the link keeps changing.

**Reply by KitsuneDev** [reply]
I see. Should it be added as a custom jar then? Or should I just put the installer in the `loaders` folder?
